### DISCUSSION TITLE
How to uppercase faker value

### DISCUSSION THREAD

**Discussion by kierangirvan** [mockoon/mockoon#654] [Help] [answered]
HI,

I need to uppercase a value generated via faker, I'm struggling to work out how I can achieve this.  The following returns a random alphanumeric value but to simulate the underlying application the response needs to be in uppercase.  As faker is no longer maintained I'm struggling to find any documentation.

`{{ faker 'random.alphaNumeric' 10 }}`

Hopefully it's an easy one to achieve.

Any help would be greatly appreciated.

**Comment by 255kb** [comment, marked as answer]
Hi @kierangirvan 
We switched to this library https://fakerjs.dev/api/random.html#alpha after Faker.js' "deprecation". We didn't implemented the latest version though. So you may take the docs with a grain of salt :) 
As you will see, there is a `upcase` option that you can use like this: 
`{{ faker 'random.alpha' count=10 upcase=true }}`
Unfortunately, there is no upcase option for `random.alphaNumeric`, and it doesn't seem to be planned. 

If you want, you can open a new issue to add `uppercase` and `lowercase` helpers. This would be useful. 

Thank you

**Reply by kierangirvan** [reply]
Thank you @255kb for your response.

This might just work for us so thank you so much for your suggestion.  I'll raise a new issue for the upper/lower case helpers.
